Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift

```diff
@@ -482,6 +495,23 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
         }
     }
 
+    private func createProductAttributeRows(rows: inout [FormRow]) {
+        if let attributeGroups = product.productAttributes?.attributeGroups {
+            for attrGroup in attributeGroups where attrGroup.id == "labels" {
+                if let attributes = attrGroup.attributes {
+                    for attribute in attributes {
+                        if let desc = attribute.descriptionShort ?? attribute.title,
+                            desc != "", let name = attribute.name, name != "" {
```

Comment:
I am filtering the attribute groups for Labels only. There are several more of them like Allergens, Nutritional Quality, etc. But since the original assignment was to create a mock-up only for the 'Organic' and 'AB bio' Labels, I have made this function specific to them.

@teolemon, do you have other attribute groups you want to display? If so, what conditions for displaying them? If not, then I will leave this as-is for the sake of just having a mock-up which won't be merged into develop.

Reply:
We don't know what the groups will be in the future. Whenever you're starting showing a group, there should be a small toggle to hide it. Ideally, we'd have a whole section of settings dedicated to enabling and disabling, and populated as new attributes groups are discovered. (if a new group is discovered, a new settings key is created where the group is enabled)

Reply:
Alright I will leave it as is for now until I get the rest of the UI parts finished, then possibly move this stuff into a 5th tab first, and then move it to summary tab, as discussed. Hopefully I will be done after this weekend, definitely by next week.